Octokit.GraphQL.nuspec

```
@@ -18,9 +18,9 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.dll" target="lib\netstandard1.1"/>
-    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.xml" target="lib\netstandard1.1"/>
-    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.Core.dll" target="lib\netstandard1.1"/>
-    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.Core.xml" target="lib\netstandard1.1"/>
+    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.dll" target="lib\netstandard2.0"/>
+    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.xml" target="lib\netstandard2.0"/>
+    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.Core.dll" target="lib\netstandard2.0"/>
+    <file src="Octokit.GraphQL\bin\Release\netstandard1.1\Octokit.GraphQL.Core.xml" target="lib\netstandard2.0"/>
```

These `src` paths need to be updated too: (I overlooked that before)

```suggestion
    <file src="Octokit.GraphQL\bin\Release\netstandard2.0\Octokit.GraphQL.dll" target="lib\netstandard2.0"/>
    <file src="Octokit.GraphQL\bin\Release\netstandard2.0\Octokit.GraphQL.xml" target="lib\netstandard2.0"/>
    <file src="Octokit.GraphQL\bin\Release\netstandard2.0\Octokit.GraphQL.Core.dll" target="lib\netstandard2.0"/>
    <file src="Octokit.GraphQL\bin\Release\netstandard2.0\Octokit.GraphQL.Core.xml" target="lib\netstandard2.0"/>
```